N/nativefiledialog_extended/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "nativefiledialog_extended"
+version = v"1.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/btzy/nativefiledialog-extended/", "5786fabceeaee4d892f3c7a16b243796244cdddc", "nativefiledialog-extended-1.1.1")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd nativefiledialog-extended-1.1.1/
```

Comment:
Again, better to avoid hard coding the version. With the change to the GitSource, I think you can simply do
```suggestion
cd nativefiledialog-extended
```